collab-service/src/redis_adapter/redis_question_adapter.ts

```diff
@@ -14,8 +16,8 @@ async function setQuestionRedis(
   question: Question,
   publisher: RedisClientType,
 ) {
-  await publisher.set(`${redisPrefix}-${key}`, JSON.stringify(question), {
-    EX: 300,
+  await publisher.set(`${REDIS_PREFIX}-${key}`, JSON.stringify(question), {
+    EX: HEARTBEAT_INTERVAL,
```

Comment:
You need to set expiry to something longer than heartbeat interval, maybe something like `3 * HEARTBEAT_INTERVAL`.